src/set_relation/set_relation_test.cc

```diff
@@ -3850,3 +3850,75 @@ TEST_F(SetRelationTest, boundDomainRange) {
     delete nr;
 }
 
+#pragma mark superAffineSet
```

Comment:
Need more and smaller test cases.  Also make sure to have plenty of nesting of uninterpreted functions and some examples with indexing, f(i)[0].  See other test cases for examples.